From 31bbdfdde271557a7c678a3fe38da4567a03c9ac Mon Sep 17 00:00:00 2001
From: Divya Narayanan Poojary <dnaray@codeaurora.org>
Date: Mon, 3 Jul 2017 11:26:29 +0530
Subject: [PATCH 5/5] tinycompress: Generate vendor specifc tinycompress

Generate libtinycompress_vendor that contains the
tinycompress extensions

CRs-Fixed: 2074491

Change-Id: I1466c13f216ce723d63155bc099eccad4fdccdfe

# Conflicts:
#	Android.mk
#	compress.c
---
 compress.c                          | 2 ++
 include/tinycompress/tinycompress.h | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/compress.c b/compress.c
index 1cdf297..6698096 100644
--- a/compress.c
+++ b/compress.c
@@ -572,6 +572,7 @@ int compress_set_gapless_metadata(struct compress *compress,
 	return 0;
 }
 
+#ifdef USE_VENDOR_EXTN
 #if defined(SNDRV_COMPRESS_SET_NEXT_TRACK_PARAM)
 int compress_set_next_track_param(struct compress *compress,
 	union snd_codec_options *codec_options)
@@ -590,6 +591,7 @@ int compress_set_next_track_param(struct compress *compress __unused,
 	return 0;
 }
 #endif
+#endif
 
 bool is_codec_supported(unsigned int card, unsigned int device,
 		unsigned int flags, struct snd_codec *codec)
diff --git a/include/tinycompress/tinycompress.h b/include/tinycompress/tinycompress.h
index f88c2a2..b7e57bc 100644
--- a/include/tinycompress/tinycompress.h
+++ b/include/tinycompress/tinycompress.h
@@ -246,8 +246,10 @@ int compress_set_gapless_metadata(struct compress *compress,
  * @codec_options: codec options of compress stream based on codec type
  */
 
+#ifdef USE_VENDOR_EXTN
 int compress_set_next_track_param(struct compress *compress,
 			union snd_codec_options *codec_options);
+#endif
 
 /*
  * is_codec_supported:check if the given codec is supported
-- 
2.17.1

